include/phasar/PhasarLLVM/DataFlowSolver/Mono/Solver/MonoCache.h

```diff
@@ -0,0 +1,64 @@
+/******************************************************************************
+ * Copyright (c) 2020 Philipp Schubert.
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of LICENSE.txt.
+ *
+ * Contributors:
+ *     Ajay Subramanya Kudli Prasanna Kumar, Philipp Schubert and others
+ *****************************************************************************/
+
+#ifndef PHASAR_PHASARLLVM_MONO_SOLVER_MONOCACHE_H_
+#define PHASAR_PHASARLLVM_MONO_SOLVER_MONOCACHE_H_
+
+#include<iostream>
+#include<unordered_map>
+#include<vector>
+
+namespace psr {
+  template <class K, class V>
+class MonoCache {
+  // TODO
+public:
+        unordered_map<K, V> summaries;
```

Comment:
Data members should be private, here.

Reply:
Updated the data members to private in commit f67db78d18d8c4d1110699e239ef4e9d945f1789